src/components/itemMediaInfo/itemMediaInfo.js

```diff
@@ -71,8 +72,9 @@ const attributeDelimiterHtml = layoutManager.tv ? '' : '<span class="hide">: </s
             const size = `${(version.Size / (1024 * 1024)).toFixed(0)} MB`;
             html += `${createAttribute(globalize.translate('MediaInfoSize'), size)}<br/>`;
         }
-        for (let i = 0, length = version.MediaStreams.length; i < length; i++) {
-            const stream = version.MediaStreams[i];
+        const streams = version.MediaStreams.sort((a, b) => itemHelper.sortTracks(a, b));
+        for (let i = 0, length = streams.length; i < length; i++) {
+            const stream = streams[i];
```

Comment:
```suggestion
        for (const stream of streams) {
```
It is shorter, but the transpilation makes it larger: `try/catch`ed iterator.

Reply:
Should we change it or not? imo both is readable and if the current implementation leads to less transpilation I don't really care what we use.